api/src/hyp3_api/util.py

```diff
@@ -11,12 +11,7 @@
 def get_granules(jobs):
     granules = set()
     for job in jobs:
-        parameters = job['job_parameters']
-        if job['job_type'] == 'RTC_GAMMA':
-            granules.add(parameters['granule'])
-        if job['job_type'] == 'INSAR_GAMMA':
-            granules.add(parameters['granule1'])
-            granules.add(parameters['granule2'])
+        [granules.add(granule) for granule in job['job_parameters']['granules']]
```

Comment:
could be one big set comprehension, I think, but don't think I like it any better
`{granule for granule in job['job_parameters']['granules'] for job in jobs}`

and this assumes every job will have a `granules` parameter, but that's fine